src/data/index.ts

```diff
@@ -22,11 +22,20 @@ export function getTokenIdentifier(token: Token): string {
  * @returns Returns `${tokenCode}:${issuerKey}`.
  */
 export function getBalanceIdentifier(balance: Horizon.BalanceLine): string {
-  if (balance.asset_type === "native" || !balance.asset_issuer) {
-    return "native";
-  }
+  switch (balance.asset_type) {
+    case "native":
+      return balance.asset_type;
+
+    case "credit_alphanum4":
+    case "credit_alphanum12":
+      return `${balance.asset_code}:${balance.asset_issuer}`;
```

Comment:
Caught this on a second pass: do we still need the `!balance.asset_issuer` check we had previously? I don't know when that comes up.

Reply:
Also, could combine `case "native"` and `default` since both return `"native"` :+1: